upgrade-instructions.md

```diff
@@ -16,17 +16,28 @@ Changes to apply between 3.4.x and 3.5.x will be reported here.
 
 Change list:
 
+ - [RabbitMQ minimal version](#rabbitmq-minimal-version)
  - [Enforce usernames to be lower cased](#enforce-usernames-to-be-lower-cased)
  - [Cassandra keyspace creation configuration](#cassandra-keyspace-creation-configuration)
  - [UsersFileRepository](#usersfilerepository)
  - [ElasticSearch performance enhancements](#elasticsearch-performance-enhancements)
  - [JAMES-2703 Post 3.4.0 release removals](#james-2703-post-340-release-removals)
 
+#### RabbitMQ minimal version
+
+Date 26/11/2019
+
+SHA-1 0bf4e8384e
+
+The distributed James project (relying on Guice, Cassandra, ElasticSearch, RabbitMQ and optionally Swift) benefits from a new distributed task mananger.
+
+For the plain operation if this new task manager, James now relies at least on RabbitMQ 3.8. So please upgrade your RabbitMQ version if you use this component.
```

Comment:
s/if/of ?

Reply:
We should not speak about components here but about products. How as a user do I know which component I run?

Reply:
Then reformulate that cause grammatically speaking, it's wrong

Reply:
if `using` this new task manager?